reVX/least_cost_xmission/offshore_utilities.py

```diff
@@ -26,18 +26,18 @@ def _sum(a, b):
     return a + b
 
 
-def convert_pois_to_lines(poi_csv_f, template_f, out_f):
+def convert_pois_to_lines(poi_csv_f: str, template_f: str, out_f: str):
     """
     Convert POIs in CSV to lines and save in a geopackage as substations. Also
     create a fake transmission line to connect to the substations.
 
     Parameters
     ----------
-    poi_csv_f : str
+    poi_csv_f
```

Comment:
Is the removal of the type in the docstring intentional? I've never seen recommendations for dropping the type in a numpy docstring, even if you do have type hints

Reply:
It is intentional and trying to keep it DRY. I didn't see a benefit to having the redundancy and certainly saw the potential for the type hint to change and the doc string to be overlooked. Grant saw the sphinx generated doc and approved it.